packages/tradershub/src/components/CFDSection/CFDContent/CFDContent.tsx

```diff
@@ -2,10 +2,9 @@ import React from 'react';
 import { TradingAppCardLoader } from '@/components';
 import { useRegulationFlags } from '@/hooks';
 import { CTraderList, MT5PlatformsList, OtherCFDPlatformsList } from '@cfd/components';
-import { useIsEuRegion } from '@deriv/api';
 
 const CFDContent = () => {
-    const { isSuccess: isRegulationAccessible } = useIsEuRegion();
```

Comment:
Reason of replacing this cause useRegulationFlags already use this hook and isEU coming from useRegulationFlags also takes account of the EU switcher condition for DIEL countries. So two birds one stone 😺